adapters/repos/db/index_integration_test.go

```diff
@@ -688,7 +713,7 @@ func TestIndex_DebugResetVectorIndexTargetVector(t *testing.T) {
 
 func TestIndex_DebugResetVectorIndexPQ(t *testing.T) {
 	t.Setenv("ASYNC_INDEXING", "true")
-	t.Setenv("ASYNC_INDEX_INTERVAL", "100ms")
```

Comment:
should we keep the old name for backward compatability?

Reply:
I'm not sure as it's not the same logic anymore and also these env vars are not part of the public API. They are primarily used for tests for now.
Once we make async indexing GA we'll communicate on which env var is supported